src/context-selection/context-selection.js

```diff
@@ -1,3 +1,32 @@
-const ContextSelection = () => null
+import { SelectorBar } from '@dhis2/ui'
+import React from 'react'
+import { CategoryOptionComboSelectorBarItem } from './category-option-combo-selector-bar-item/index.js'
+import { DataSetSelectorBarItem } from './data-set-selector-bar-item/index.js'
+import { OrgUnitSetSelectorBarItem } from './org-unit-selector-bar-item/index.js'
+import { PeriodSelectorBarItem } from './period-selector-bar-item/index.js'
+import { SectionFilterSelectorBarItem } from './section-filter-selector-bar-item/index.js'
+import { useContextSelection } from './use-context-selection.js'
 
-export default ContextSelection
+export default function ContextSelector() {
+    const [, setSelectionContext] = useContextSelection()
+
+    const onClearSelectionClick = () => {
```

Comment:
Maybe you could create a hook `useClearContextSelection` and colocate that with the definition of the query param schema. Means code outside of that module doesn't need to know about the schema.